documents/operating.md

```diff
@@ -13,10 +13,44 @@ Contents:
 ---------------------------------------------------------------------
 SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
 ---------------------------------------------------------------------
-- [ ] Company term
-- [ ] Continuance of company
-- [ ] Rickroll
-- [ ] Game of Mario Party 3 before legal arbitration between members
+- [x] Company term
+- [x] Continuance of company
+- [x] Rickroll
+- [x] Game of Mario Party 3 before legal arbitration between members
+
+### Company Term
+BIFFUD will exist until either (1) a vote to terminate the organization
+is passed by a consensus of all Corporate Overlords or (2) the board fails to
+hold a single meeting with quorum for a period of (a consecutive year?),
+(3) everyone dies / there are no more Corporate Overlords, or (4) if all
+Corporate Overlords are straight white cis men.
+
+### Continuance of Company
+If a Corporate Overlord leaves BIFFUD, dies, or uploads their brain to the
+cloud, the ownership of the remaining corporate overlords will then be evenly
+distributed (e.g. if four overlords remain, each will own 25% of BIFFUD)
+
+### Lawsuits between BIFFUD associates
+All BIFFUD associates involved in a lawsuit must play a group game of
+Mario Party 3 before any arbitration happens.  BIFFUD will loan any
+necessary hardware needed to make this possible.
+
```

Comment:
Does the console need to be specified? It only legally exists on N64 right now, but what if it is re-released on some other medium in the future?